lib/faker/phone.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Faker.Phone do
+  @moduledoc """
+  Functions for generating phone numbers
+  """
+
+  @doc """
+  Return a random United States phone number
+  """
+  @spec us_phone(separator :: String.t) :: String.t
+  def us_phone(separator \\ "-") do
```

Comment:
lets split it like in ruby faker:
```
Faker::PhoneNumber.area_code #=> "201"
Faker::PhoneNumber.exchange_code #=> "208"
Faker::PhoneNumber.subscriber_number #=> "3873"
Faker::PhoneNumber.subscriber_number(2) #=> "39"
Faker::PhoneNumber.extension #=> "3764"
```
To have ability to use them separately, if someone have `area_code` as different field in db for example.